css/file

```diff
@@ -0,0 +1 @@
+
```

Comment:
Empty file?  Can this be removed?

Reply:
Yes will remove all (was for folder making)

Reply:
All removed